google/ggrc-core
# GGRC-6727 Properly handle review_status_display_name column for Control

# Issue description

`review_status` column is exportable instead of `review_status_display_name`. `review_status_display_name` is not populated during restore from revisions. "Review State" name is used for `review_status_display_name` column instead of "Review Status".

# Steps to test the changes

Ensure that `review_status_display_name` is set to `review_status` value when restoring Control. Ensure that control are searchable by "Review Status" column which is mapped to `review_status_display_name`.

# Sanity checklist




# PR Review checklist
